### DISCUSSION TITLE
Doris BE机器损坏后无法正常下线

### DISCUSSION THREAD

**Discussion by MountHuang** [apache/doris#12325] [A - General / Q&A]
Doris集群中一台BE机器突然硬件损坏后，无法通过正常途径decommission，只好将它强行drop掉，但存储于该节点之上的tablet REPLICA虽然处于DEAD的状态，迟迟不从列表中消失，而且其它节点同步该tablet也会报version error. 请问大家有遇到过这样的问题吗？是否有解决的思路？ 感谢！

（版本0.14.0）
<img width="1522" alt="截屏2022-09-04 下午7 20 01" src="https://user-images.githubusercontent.com/2619346/188311432-a8c1a2db-e1e5-400e-9173-6d571bdc9967.png">

**Comment by yuanyuan8983** [comment]
你好，首先看你的副本数是否是多副本的情况，如果是多副本的情况，那么可以试试把出问题的Tablet设置为bad，然后让剩余BE知道这个Tablet坏掉了，可以先测试一下，可以参考官网参考下https://doris.apache.org/zh-CN/docs/admin-manual/maint-monitor/tablet-repair-and-balance?_highlight=副本#副本均衡

**Reply by MountHuang** [reply]
感谢提供信息，只是我们目前已经将出问题的Tablet设置为bad了，但好像没有达到预期的作用，其它BE在clone这个tablet的时候还是clone的错误的那个。在查看tablet详细信息的时候也开始报unknown error，请问还有其它办法让剩余BE知道这个Tablet坏掉了吗？

<img width="712" alt="iShot2022-09-05 13 53 03" src="https://user-images.githubusercontent.com/2619346/188369903-814f1398-90e7-4a53-bcfd-feb9da2ea937.png">

上面那个Unknown error对应的日志是这样的
<img width="1461" alt="iShot2022-09-05 13 56 47" src="https://user-images.githubusercontent.com/2619346/188370340-df5dec71-719c-4c03-b85a-a8cc5268f4e5.png">

**Comment by hf200012** [comment]
如果你是多副本的，你可以直接使用alter system dropp backend 命令将这个节点从集群中删除，等修复好之后再作为一个新的节点重新加入到集群中

**Reply by MountHuang** [reply]
是多副本的，我们已经做过这样的尝试了，但这个机器上损坏的副本信息依然遗留在集群里，无法被剔除
